<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <property name="spacing">
     <number>4</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Explore</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QComboBox" name="comboBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QTreeView" name="treeView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Setting</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="SettingsWidget" name="widget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>C3</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="JSWidget" name="widget_3" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>MySetting</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="MySettingWidget" name="widget_2" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Test Widget</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>12</y>
          <width>125</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>20</y>
          <width>125</width>
          <height>21</height>
         </rect>
        </property>
        <property name="inputMask">
         <string>dddd/dd/dd</string>
        </property>
        <property name="text">
         <string>212/1/2</string>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidget">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>96</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::AllEditTriggers</set>
        </property>
        <property name="viewMode">
         <enum>QListView::ListMode</enum>
        </property>
        <item>
         <property name="text">
          <string>abc</string>
         </property>
        </item>
       </widget>
       <widget class="QListView" name="listView">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>90</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::AllEditTriggers</set>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>50</y>
          <width>175</width>
          <height>24</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 0, 255)</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>60</y>
          <width>87</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>310</y>
          <width>91</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="tool.qrc">:/mytool/img/face-smile 2.png</pixmap>
        </property>
       </widget>
       <widget class="QToolButton" name="btnCalendar">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>310</y>
          <width>26</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_2">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>40</y>
          <width>104</width>
          <height>21</height>
         </rect>
        </property>
        <property name="maxVisibleItems">
         <number>11</number>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <property name="spacing">
         <number>4</number>
        </property>
        <item row="0" column="0">
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_7">
           <attribute name="title">
            <string>Tab 1</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="DrawWidget" name="widget_4" native="true"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_8">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_8">
            <property name="leftMargin">
             <number>4</number>
            </property>
            <property name="topMargin">
             <number>4</number>
            </property>
            <property name="rightMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>4</number>
            </property>
            <property name="spacing">
             <number>2</number>
            </property>
            <item row="0" column="0">
             <widget class="MyGraphicsView" name="graphicsView"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>608</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionEDC"/>
    <addaction name="actionSplit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionEDC">
   <property name="text">
    <string>EDC</string>
   </property>
  </action>
  <action name="actionSplit">
   <property name="text">
    <string>Split</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SettingsWidget</class>
   <extends>QWidget</extends>
   <header>settingswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MySettingWidget</class>
   <extends>QWidget</extends>
   <header>mysettingwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>JSWidget</class>
   <extends>QWidget</extends>
   <header>jswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DrawWidget</class>
   <extends>QWidget</extends>
   <header>drawwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>mygraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="tool.qrc"/>
 </resources>
 <connections/>
</ui>
